src/app/personside/infotabs/saksoversikt/viktigavite/DAGInnhold.tsx

```diff
@@ -9,7 +9,7 @@ export function ViktigÅViteDAGInnhold(): JSX.Element {
                 Vi forventer at du selv aktivt forsøker å skaffe deg jobb. Du må derfor søke jobber. Dersom du ikke
                 finner ledige stillinger innenfor det du ønsker å arbeide med, må du utvide jobbsøket ditt. Det gjør du
                 selv på Ditt NAV.&nbsp;
-                <a href="https://tjenester.nav.no/stillinger/" target="_blank">
+                <a href="https://tjenester.nav.no/stillinger/" target="_blank" rel="noopener noreferrer">
```

Comment:
Kan evt. vurdere `Lenke` komponenten fra nav-frontend. Den skal håndtere `rel`-attributten for deg

Reply:
Lurt! Jeg bare fiksa gammel kode pga eslint-warnings (dette ble merget inn i annen PR)